src/delete.js

```diff
@@ -0,0 +1,40 @@
+const todolist = JSON.parse(localStorage.getItem('list')) || [];
+export const forms = document.querySelector('.todo-form');
+export const input = document.querySelector('.todo-input');
+export const clear = document.querySelector('.submit');
+
+const deletefun = () => {
```

Comment:
- Please Implement a function for deleting a task (remove an element from the array): Kindly make sure that you are able to delete a selected book.
- Your App can only clear All. please fix it.
![Capture](https://user-images.githubusercontent.com/63972670/193176313-c0559b80-1f26-4f1a-a631-56d3a8379b7d.PNG)
- Please, After implementing the change, make sure that Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. Also, all changes to the To-Do List should be saved in local storage.